### ISSUE TITLE
历史命令无法显示

### ISSUE THREAD

**Issue by LuckX** [kingToolbox/WindTerm#1167]
windows10 64环境使用2.6.0版本，不显示折叠，不显示历史命令，命令无联想

**Comment by LuckX** [comment]
只有Windows powershell会话有，cmd 和 linux会话这几个功能都没有

**Comment by liujunhui1** [comment]
![image](https://user-images.githubusercontent.com/33777600/188053521-9fe9acc0-0207-419f-a9cb-50db7c6fb0af.png)
这个选了吗

**Comment by LuckX** [comment]
> ![image](https://user-images.githubusercontent.com/33777600/188053521-9fe9acc0-0207-419f-a9cb-50db7c6fb0af.png) 这个选了吗

都选了

**Comment by wangxiaojuaner** [comment]
> > ![image](https://user-images.githubusercontent.com/33777600/188053521-9fe9acc0-0207-419f-a9cb-50db7c6fb0af.png) 这个选了吗
> 
> 都选了

我指导问题了，ssh连接要选择linux，选cmd是没有的。
![image](https://github.com/kingToolbox/WindTerm/assets/51771488/78f29b37-554c-4ec2-8820-0a8f455735cd)

**Comment by doulongfei** [comment]
有没有人和我一样的问题，需要输入一个空格，然后在敲命令，就会记录历史命令的？

**Comment by sduzgx** [comment]
用bash可以，zsh不行

**Comment by qcq** [comment]
> 用bash可以，zsh不行

that is right, seems windterm used .bash_history file to reference the history cmd, have to check the source code to make sure this, BTW, zsh used .zsh_history file to store that history cmd

**Comment by ji-eking** [comment]
mac ios 26 完全按上图设置，也不能保存历史记录

**Comment by doveppp** [comment]
PS1 环境变量，也就是终端格式，最后要有个空格
